.travis_scripts/build-release.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -e
+set -x
+
+# JAVA_HOME isn't set on OSX for some reason, so manually set it
+if [ -z "$JAVA_HOME" ]; then
```

Comment:
Can we do this at the top level so it doesn't have to be duplicated in both shell scripts?

Reply:
Don't think you can do if-y statements like this when setting the cross shell env stuff in travis. I'm certainly no expert though. Doesn't feel like something worth spending the time to figure out though.